content/docs/multi-tenancy/getting-started.md

```diff
@@ -1,79 +1,28 @@
 +++
```

Comment:
Because we're effectively deleting a page, we need to take care of links pointing to the original location of the page:
* [Search for](https://github.com/kubeflow/website/search?q=%22multi-user-overview%22&unscoped_q=%22multi-user-overview%22) and fix all links within the website that point to the moved page.
* Put a redirect in place, to capture all requests coming in from outside the docs. To implement a redirect, add a line to this file:
https://github.com/kubeflow/website/blob/master/content/_redirects